emperor/support_files/js/abc-view-controller.js

```diff
@@ -183,5 +183,14 @@ define([
     throw Error('Not implemented');
   };
 
+  /**
+   * Writes the current settings to the active decomposition view(s).
```

Comment:
- Looks like save to PNG is not working with parallel plots.

Reply:
Done.  Presumably CanvasRenderer can't handle shaders, so I set it to use the other renderer, same as you did for large point cloud datasets.

Reply:
NOTE:  I have no way to test the svg mode.  I can't even get svg mode to work correctly on scatter plots on my machine.